spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/ops/Equal.scala

```diff
@@ -0,0 +1,209 @@
+/*
+ * Copyright 2016 The BigDL Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.intel.analytics.bigdl.nn.ops
+
+import com.intel.analytics.bigdl.tensor._
+import com.intel.analytics.bigdl.tensor.TensorNumericMath.TensorNumeric
+import com.intel.analytics.bigdl.utils.Table
+
+import scala.reflect.ClassTag
+
+class Equal[T: ClassTag]()
+  (implicit ev: TensorNumeric[T]) extends Operation[Table, T] {
+
+  /**
+   * Iterate through tensor1, tensor2, and apply func to the elements,
+   * set function result to tensor 3
+   *
+   * @param tensor1 the tensor1
+   * @param tensor2 the tensor2
+   * @param tensor3 the result tensor
+   * @param func    (tensor1Data, tensor1Offset, tensor2Data,
+   *                tensor2Offset, tensor3Data, tensor3Offset)
+   */
+  def apply2[A, B, C](tensor1: Tensor[A], tensor2: Tensor[B], tensor3: Tensor[C],
```

Comment:
How about move this method to a until object in ops package, so that other Operation can also use it?